.github/workflows/codecov_carryforward_reports.yml

```diff
@@ -0,0 +1,45 @@
+name: Carry forward codecov reports
```

Comment:
This job is fairly inexpensive. It takes about ~2min to run

See examples: https://github.com/giovanni-guidini/sentry/actions/workflows/codecov_carryforward_reports.yml